nixos/modules/programs/light.nix

```diff
@@ -4,6 +4,7 @@ with lib;
 
 let
   cfg = config.programs.light;
+  light = if cfg.devicePath != null then "${pkgs.light}/bin/light -s ${cfg.devicePath}" else "${pkgs.light}/bin/light";
```

Comment:
I'd do something like the following (untested):

```nix
light = "${pkgs.light}/bin/light" + lib.optionalString (cfg.devicePath != null) " -s ${cfg.devicePath}";
```

Reply:
@Synthetica9 Applied, thanks!